### PR TITLE
adds methos (mentos)

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
it makes foam with cola, fun for ghetto people, the candy contains meth ingredients because pun

:cl: fikou did the name
add: methos, a new contaband candy, dont mix with space cola!
/:cl:

### CODE REVIEW

**Comment by Shadowflame909** [comment]
I know a gamer who put 100 mints into his mouth and then put cola into his mouth, and he lost a tooth

File: code/modules/reagents/reagent_containers/pill.dm [hunk 1]
```diff
@@ -275,3 +275,11 @@
 	icon_state = "pill8"
 	list_reagents = list(/datum/reagent/iron = 30)
 	rename_with_volume = TRUE
+
+/obj/item/reagent_containers/pill/methos
+	name = "Methos(TM)"
+	desc = "A sugary threat."
+	icon_state = "methos"
+	list_reagents = list(/datum/reagent/drug/methamphetamine = 1, /datum/reagent/consumable/mentonol = 10)
```
Comment (nemvar) [inline code/modules/reagents/reagent_containers/pill.dm:283]: Why are you like this?

Comment (Tlaltecuhtli) [reply code/modules/reagents/reagent_containers/pill.dm:283]: whats wrong?

Comment (nemvar) [reply code/modules/reagents/reagent_containers/pill.dm:283]: Why don't you put random packs of bath salts in maint while you are at it.
I want to know what's right about this first. Meth is *essentially* a lategame chem which isn't that easily aquired without chemical equipment or botany.
You can do a lot of shitty stuff with even small quantities of meth and the only other way of aquiring it this easily is with addict quirk, which has obvious drawbacks and is RNG dependent.
The pun is also really bad.
I wouldn't mind if this was something you can use as pseudo to cook up ghetto meth but just plain meth invalidates too many other tasks.

Comment (Tlaltecuhtli) [reply code/modules/reagents/reagent_containers/pill.dm:283]: is 0.4 meth ok? (1 tick) and meth isnt a lategame chem, just look at shit like rdx, tatp, life and then look at meth made without ever swapping a beaker and an igniter

Comment (optimumtact) [reply code/modules/reagents/reagent_containers/pill.dm:283]: why are you like this

**Comment by optimumtact** [comment]
Take the meth out you dip

File: code/modules/reagents/reagent_containers/pill.dm (hunk 1 above)
Comment (Tlaltecuhtli) [reply code/modules/reagents/reagent_containers/pill.dm]: i ll go for "I wouldn't mind if this was something you can use as pseudo to cook up ghetto meth " suggestion

**Comment by derpymcfuckwit** [comment]
what the fuck this is a comedic masterpiece lmao

**Comment by HomocidlePancakes** [comment]
Do not confirm. Keep the meth in.

**Comment by Moja10** [comment]
Replace with pump up V2 where the heartbeats are louder and actually raise your heartrate IRL and only the powergamers will be able to tell the different with PUV2 and actual meth

**Comment by optimumtact** [comment]
thog not care

**Pull request closed without merging**